### PR TITLE
fix shadow cut

### LINKED ISSUE #4251: [NV-2040] 👁️ [Visual fix] Fix cut shadow
## What?
On the Get Started page in the second step, the shadow is cut at the bottom of the Email node.

image

## Why?
Visually it doesn't look good :)

## Definition of done
- make the react flow container height or min-height a little bigger
- on the 1280x720 there should be no scrollbar

### LINKED ISSUE #4266: 🐛 Bug Report: [Quick start guide] Getting started page : React flow button UI fixing.
### 📜 Description

When we visit the getting started page , the React Flow anchor tag link is not clearly visible finds makes user experience unsatisfactory.
Screenshot 2023-10-01 091452

Color overlaps with the element behind and two links are placed very close to each other which does not look good.

### 👟 Reproduction steps

1. Go to the Getting started page.
2. Look down for the Element having a title 'Channel' and description 'Build desired order of channels' .
3. In the same element there is a pink colored link which looks great .
4. Below that you will see the React Flow link which is not clearly visible.
Screenshot 2023-10-01 091452

### 👍 Expected behavior

The React Flow link which redirects to [https://reactflow.dev/](url) should have seperate position and should be highlighted either with a lighter shade or a different color. The link should appear distant from background element.

### 👎 Actual Behavior with Screenshots

Screenshot 2023-10-01 091452

### Novu version

Novu Saas

### npm version

_No response_

### node version

_No response_

### 📃 Provide any additional context for the Bug.

_No response_

### 👀 Have you spent some time to check if this bug has been raised before?

- [X] I checked and didn't find a similar issue

### 🏢 Have you read the Contributing Guidelines?

- [X] I have read the [Contributing Guidelines](https://github.com/novuhq/novu/blob/main/CONTRIBUTING.md)

### Are you willing to submit PR?

Yes I am willing to submit a PR!

### PR DESCRIPTION
**Pull request by jatinder14**
### What change does this PR introduce?

it remove the shadow cut and make the columns align 
also it will set the position of React flow element

### Why was this change needed?
Visually beautifies the page  :)
and fix the overlap of react flow element

Closes #4251 
Closes #4266

### CODE REVIEW

**Comment by jatinder14** [comment]
@scopsy  isn't my PR fix the react flow div element?

File: apps/web/src/components/workflow/NodeStep.tsx [hunk 1]
```diff
@@ -57,7 +57,7 @@ const StepCard: any = styled.div`
   position: relative;
   display: flex;
   width: 280px;
-  height: 80px;
+  height: 105px;
```
Comment (LetItRock) [inline apps/web/src/components/workflow/NodeStep.tsx:60]: we don't want to change the workflow node size

File: apps/web/src/components/workflow/FlowEditor.tsx [hunk 2]
```diff
@@ -375,6 +375,7 @@ const Wrapper = styled.div<{ dark: boolean }>`
   .react-flow__attribution {
     background: transparent;
     opacity: 0.5;
+    right: 133px;
```
Comment (LetItRock) [inline apps/web/src/components/workflow/FlowEditor.tsx:378]: this change doesn't fix the attribution problem

**Review by LetItRock — CHANGES_REQUESTED** [review]
hey @jatinder14 ! 👋 
Thanks for your contribution and effort. But unfortunately the fix that you've provided doesn't solve any of the two issues that you've linked. Please check the images attached.

1. We don't want to change the workflow node height (screenshot 1).
2. The "React Flow" attribution is still not positioned correctly.
3. The shadow is still cut (screenshot 2).

Screenshot 2023-10-02 at 17 14 13
Screenshot 2023-10-02 at 17 17 02

File: apps/web/src/components/workflow/FlowEditor.tsx (hunk 2 above)
Comment (jatinder14) [reply apps/web/src/components/workflow/FlowEditor.tsx:378]: @chankruze sorry for what I fixed
But would you please tell me the correct way to solve it .?
I literally want to learn new things.
And one thing more there is no tshirt in this year

Comment (chankruze) [reply apps/web/src/components/workflow/FlowEditor.tsx:378]: If you don't know how to fix then why you asked to be assigned and false claimed that you have already fixed those issues.

If you want to learn then fork play with it locally and silently.

I have checked your history (no contribution) but suddenly in October you decided to contribute by raising PR with shit code just to get it merged and get some swaggs.

Those Hacktobefest and Hacksquad are encouraging to learn and contribute what you are capable of. Contribution to any OSS project means understanding, interpreting and solving issue in the correct way following repo guidelines. NOT to bark "assign me assign me" having no idea of the solution.

Please, first learn then jump into this shit.

Comment (jatinder14) [reply apps/web/src/components/workflow/FlowEditor.tsx:378]: People like you are a curse for the planet earth @chankruze

Comment (chankruze) [reply apps/web/src/components/workflow/FlowEditor.tsx:378]: People (uneducated noobs who try to contribute for the shake of swags) like you are curse for open source.

Comment (unicodeveloper) [reply apps/web/src/components/workflow/FlowEditor.tsx:378]: @jatinder14 We maintain respectful communication in this community. Please delete/retract your comment.

**Comment by LetItRock** [comment]
hey @jatinder14 👋 
have you had a chance to take a look at my suggestions?

**Comment by LetItRock** [comment]
Closing this one as the author did not implement the suggestions.

**Pull request closed without merging**